src/System.Runtime.Serialization.Xml/tests/XmlDictionaryWriterTest.cs

```diff
@@ -42,26 +42,51 @@ public static void XmlBaseWriter_WriteBase64Async()
     [Fact]
     public static void XmlBaseWriter_FlushAsync()
     {
-        string actual;
+        string actual = null;
         int byteSize = 1024;
         byte[] bytes = GetByteArray(byteSize);
         string expect = GetExpectString(bytes, byteSize);
-        using (var ms = new AsyncMemoryStream())
+        var messageBuilder = new StringBuilder();
+        try
+        {            
+            using (var ms = new AsyncMemoryStream())
+            {
+                var writer = XmlDictionaryWriter.CreateTextWriter(ms);
+                messageBuilder.AppendLine("XmlDictionaryWriter.CreateTextWriter() completed.");
+
```

Comment:
No need to keep appending to a message builder. You can have a string and just change the value after each step.